components/service/glean/src/test/java/mozilla/components/service/glean/storages/EventsStorageEngineTest.kt

```diff
@@ -4,18 +4,39 @@
 package mozilla.components.service.glean.storages
 
 import android.os.SystemClock
+import androidx.test.core.app.ApplicationProvider
+import mozilla.components.service.glean.checkPingSchema
+import mozilla.components.service.glean.config.Configuration
+import mozilla.components.service.glean.FakeDispatchersInTest
+import mozilla.components.service.glean.Glean
+import mozilla.components.service.glean.EventMetricType
+import mozilla.components.service.glean.Lifetime
+import mozilla.components.service.glean.net.HttpPingUploader
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.json.JSONObject
 import org.junit.Before
 import org.junit.Test
 import org.junit.Assert.assertEquals
 import org.junit.Assert.assertNull
+import org.junit.Rule
 import org.junit.runner.RunWith
 import org.robolectric.RobolectricTestRunner
 
 @RunWith(RobolectricTestRunner::class)
 class EventsStorageEngineTest {
 
+    @get:Rule
+    val fakeDispatchers = FakeDispatchersInTest()
```

Comment:
This is, I think, the key to making the tests work.  This PR adds some `Dispatchers.API.launch` calls to this file for the first time (in the use of `EventMetricType.record`), and therefore needs the shim to make that stuff synchronous.

Reply:
I think so too. But that should have made everything sync: at least I'm fairly sure that was working for other tests.